pkgs/build-support/compress-drv/default.nix

```diff
@@ -0,0 +1,63 @@
+/*
+  compressDrv compresses files in a given derivation.
+
+  Inputs:
```

Comment:
There is some convention that we agreed, you can find it for example here:

https://github.com/NixOS/nixpkgs/blob/master/lib/attrsets.nix

Reply:
Updated.